### PR TITLE
add api 2.2 warning on readme

### PR DESCRIPTION
**Pull request by JakeMaldonado**
#### Why?
The sdk doesn't support API V2.2

#### How?
Updated the readme

### CODE REVIEW

**Comment by JakeMaldonado** [comment]
@apassant could you review this please

**Comment by JakeMaldonado** [comment]
@lloydliyu hey would you mind reviewing this please 🙏? There should be a warning to developers that this library doesn't work for the new API.

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 [![npm version](https://badge.fury.io/js/intercom-client.svg)](http://badge.fury.io/js/intercom-client)
 
+### Does not currently support API V2.2
```
Comment (lloydliyu) [inline README.md:8]: It is my understanding that currently only the `Customer Search` binding is available from version 2.x of the API. The rest of bindings expect version 1.0. Would `Excluding Customer Search (2.0), only supports 1.0` sound better here? Let me know what you think.

Comment (JakeMaldonado) [reply README.md]: Yeah that is better. Thanks! made the update.

**Pull request merged**